### PR TITLE
DOCSP-5703 - Remove Atlas Free Tier tutorial and link to Atlas docs.

### PR DESCRIPTION
**Pull request by jdestefano-mongo**
**JIRA:** https://jira.mongodb.org/browse/DOCSP-5703
**Staging:**

- https://docs-mongodbcom-staging.corp.mongodb.com/docsworker-xlarge/DOCSP-5703/introduction.html
- https://docs-mongodbcom-staging.corp.mongodb.com/docsworker-xlarge/DOCSP-5703/tutorial/getting-started.html#set-up-your-own-deployment

**Build log:** https://workerpool-boxgs.mongodbstitch.com/pages/job.html?jobId=5d9e67645a73421db26e6bb0

```
source/reference/privilege-actions.txt:616: WARNING: duplicate object description of "setParameter", other instance in /home/docsworker-xlarge/docs/build/DOCSP-5703/source/reference/configuration-options.txt

```

I tested the redirect changes by adding them to `master` and running `mut-publish`. The following redirects were generated in the .htaccess file: 

```
Redirect 301 /v3.4/tutorial/atlas-free-tier-setup/ https://docs.atlas.mongodb.com/getting-started/
Redirect 301 /v3.6/tutorial/atlas-free-tier-setup/ https://docs.atlas.mongodb.com/getting-started/
Redirect 301 /v4.0/tutorial/atlas-free-tier-setup/ https://docs.atlas.mongodb.com/getting-started/
Redirect 301 /manual/tutorial/atlas-free-tier-setup/ https://docs.atlas.mongodb.com/getting-started/
Redirect 301 /v4.2/tutorial/atlas-free-tier-setup/ https://docs.atlas.mongodb.com/getting-started/
Redirect 301 /manual/tutorial/atlas-free-tier-setup/ https://docs.atlas.mongodb.com/getting-started/
Redirect 301 /v4.2/tutorial/atlas-free-tier-setup/ https://docs.atlas.mongodb.com/getting-started/
Redirect 301 /master/tutorial/atlas-free-tier-setup/ https://docs.atlas.mongodb.com/getting-started/
```

This file existed from `v3.4` on. Let me know if you think I should just redirect all versions `[*]`

### CODE REVIEW

File: source/tutorial/getting-started.txt [hunk 1]
```diff
@@ -252,8 +252,8 @@ To set up your own deployment:
    * - MongoDB Atlas Free Tier Cluster
     
      - MongoDB Atlas is a fast, easy, and free way to get started with
-       MongoDB. Follow the :ref:`create-free-tier-manual` tutorial to
-       get started with MongoDB Atlas.
+       MongoDB. To learn more, see the
```
Comment (jdestefano-mongo) [inline source/tutorial/getting-started.txt]: We follow the style "To learn more, see ____", but if you prefer "For more information, see" just let me know.

File: config/redirects [hunk 2]
```diff
@@ -1621,3 +1621,5 @@ raw: /v2.8/release-notes/2.8-changes -> ${base}/v3.0/release-notes/3.0
 [v3.0-v4.0]: /${version}/reference/operator/aggregation/degreesToRadians -> ${base}/${version}/reference/operator/aggregation
 [v3.0-v4.0]: /${version}/reference/operator/aggregation/radiansToDegrees -> ${base}/${version}/reference/operator/aggregation
 
+# Redirect manual's Atlas tutorial to Atlas docs DOCSP-5703
+[v3.4-*]: /${version}/tutorial/atlas-free-tier-setup/ -> https://docs.atlas.mongodb.com/getting-started/
```
Comment (jdestefano-mongo) [inline config/redirects]: Let me know if we should just do `[*]`

**Comment by jdestefano-mongo** [comment]
@kay-kim Can you review when you have some bandwidth?

File: config/redirects (hunk 2 above)
Comment (kay-kim) [reply config/redirects]: What you have is perfect.

File: source/tutorial/getting-started.txt (hunk 1 above)
Comment (kay-kim) [reply source/tutorial/getting-started.txt]: This is fine.

File: source/introduction.txt [hunk 3]
```diff
@@ -133,7 +133,7 @@ third parties to develop storage engines for MongoDB.
    :hidden:
 
    /tutorial/getting-started
-   /tutorial/atlas-free-tier-setup
+   Create an Atlas Free Tier <https://docs.atlas.mongodb.com/getting-started/>
```
Comment (kay-kim) [inline source/introduction.txt]: Just wondering.  Do we even want this in the toc? or should we just get rid of it all together. I think it made more sense around 3.4 when Atlas was starting out.   We still benefit from better SEO but not sure,

Comment (jdestefano-mongo) [reply source/introduction.txt]: @gilberto-mdb What do you think?

**Pull request merged**